content/02.main-text.md

```diff
@@ -432,7 +432,14 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 
 Concepts for the future of scholarly publishing extend beyond collaborative writing [@doi:10.22541/au.148769949.92783646; @doi:10.22541/au.149693987.70506124; @tag:techblog-brown].
 Pandoc Scholar [@doi:10.7717/peerj-cs.112] and Bookdown [@doi:10.1201/9781315204963], which has been used for open writing [@doi:10.12688/f1000research.16516.1], both enhance traditional Markdown to better support publishing.
-The [knitcitations](https://github.com/cboettig/knitcitations) package enables citation by DOI or URL in R Markdown documents.
+[Dokieli](https://dokie.li/) stores articles as HTML and includes a clientside editor to allow reader annotation and interaction [@doi:10.1007/978-3-319-60131-1_33].
```

Comment:
We could optionally add something about it being "decentralized" here.

Reply:
It was not entirely clear to me how dokieli was decentralized in ways other authoring / publishing tools were not. It seemed that most decentralization came from the clientside editor such that any reader could "fork" a manuscript and begin editing or annotating it. Therefore, I stressed that aspect in the sentence.

Reply:
*For example*, dokieli makes it possible for a user to perform "save" or "save as" operations (request) of their article to any HTTP URI. Same goes for annotations and notifications. Subject to authentication and authorization.

That means that the user has a good degree of control to:
* Decide on and register identifiers (at any granularity) for their artifacts.
* Shape their information using machine-readable (Linked Data) vocabularies.

So, there is quite a lot more going on than storing articles in HTML. In fact, dokieli itself does not store articles. Servers do. The application is decoupled from servers, and so it is interoperable with servers that conform to the same standards (on the Open Web Platform). The data dokieli generates can be reusable by other applications conforming to the same vocabularies. See https://csarven.ca/linked-research-decentralised-web#decentralised-linked-research-application for further details.

How are the other authoring / publishing tools decentralised and/or interoperable? Which mechanisms do they use?

Reply:
Would the following sentence more clearly capture the decentralization point:

> Dokieli is a clientside editor for HTML articles, which aims for decentralization by adhering to web standards that allow articles and reader annotations to be stored by any participating server.

I agree @csarven that the ability to archive / redistribute / modify / annotate articles without permission or technical barriers is an important part of decentralization. I agree most current journals do not provide this, although many basic authoring systems potentially do. For example, if authors circulate a DOCX file with an open license, that technically satisfies the property that others can annotate, edit, and redistribute it.

I think the real decentralization benefits will come when we journals that uphold these aspects. Systems that provide immutable persistent publication with permissionless reader interaction would enable journals like this. Currently, blockchains with secure consensus mechanisms seem to come closest to a database that would support decentralized and trust-minimizing storage and interaction with scholarly articles.

Anyways... this is a digression. Let's settle on a sentence describing Dokieli!

Reply:
How about:

dokieli is a clientside editor for decentralised article publishing, annotations and social interactions by way of applying open Web standards - such as Linked Data design principles - in order to identify, describe and discover information in an interoperable way.

Reply:
I am going to stick with the phrasing in https://github.com/greenelab/meta-review/pull/255/commits/d47766e279bbd95b09f3267390682149b291dc6b.

> in order to identify, describe and discover information in an interoperable way

this sounds nice, but I think it leaves the reader thinking how? It's a non-falsifiable statement that could be said about most publishing platforms. I'd rather keep the sentenced focused on the specific functionality of different projects, and keep to a similar tone throughout the paragraph.

Thanks for the suggestions however. It has been cool to learn more about dokieli.

Reply:
Your call in the end.

I just felt that the phrasing in https://github.com/greenelab/meta-review/commit/d47766e279bbd95b09f3267390682149b291dc6b understated dokieli. That is not particularly more "falsifiable" than the one I've proposed.

"interoperability" has a very specific meaning in context of open Web standards and especially in context of Linked Data design principles. That is in fact entirely falsifiable - every single point pertaining to identification, description and discovery - and there are several arguments put forward (with supporting data) at https://csarven.ca/linked-research-decentralised-web#decentralised-linked-research-application which backs all that up. I believe.

On the other hand, it is trivial to prove how alternative solutions out there are either non-interoperable or simply implement proprietary designs relying on post-facto interpo (if at all!). "Interop" is not to be conflated with "post-facto interop".

There is not much one can state with a one sentencer.. so, I suppose it depends a bit what you want to focus on and to whom you want to communicate. Possibly consider making more than one statement if you want to sufficiently differentiate the tools and highlight the underlying approaches.